public/app/plugins/datasource/prometheus/querybuilder/shared/LabelFilterItem.tsx

```diff
@@ -98,7 +99,7 @@ export function LabelFilterItem({
             aria-label={selectors.components.QueryBuilder.matchOperatorSelect}
             value={toOption(item.op ?? defaultOp)}
             options={operators}
-            width="auto"
+            width={14}
```

Comment:
Here specifically - I personally like `auto` option more than `14`, cause it reads/looks like query. But that's just me. It is worth checking with logs and metrics team (as this is going to change UI for prometheus as well) what they think. 
`auto`: 
<img width="625" alt="image" src="https://user-images.githubusercontent.com/30407135/223158610-86197cd9-f42e-467b-be2b-695a7b16a4b7.png">
<img width="560" alt="image" src="https://user-images.githubusercontent.com/30407135/223158644-99644ccb-77d2-4468-a066-c76978ab11f2.png">

`14`:
<img width="618" alt="image" src="https://user-images.githubusercontent.com/30407135/223158923-f537e589-daeb-493f-beff-4f98b182b46e.png">
<img width="540" alt="image" src="https://user-images.githubusercontent.com/30407135/223158970-917bd43a-6be1-455f-837e-f87e445ac67b.png">